docs/source/examples/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -5,15 +5,66 @@
    "id": "104dbd2d",
```

Comment:
<blockquote>will likely make the model to be prone to errors</blockquote><p>Better write</p><blockquote>will likely make the model more prone to errors</blockquote><p>Also second sentence kinda duplicates the first, and does it better. Maybe just leave a variation of the second sentence </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>